src/concrete/ml/quantization/quantized_module.py

```diff
@@ -445,14 +454,19 @@ def _clear_forward(self, *q_x: numpy.ndarray) -> numpy.ndarray:
             layer_results[output_name] for output_name in self.ordered_module_output_names
         )
 
-        assert_true(len(output_quantized_arrays) == 1)
-
         # The output of a graph must be a QuantizedArray
-        assert isinstance(output_quantized_arrays[0], QuantizedArray)
+        assert all(isinstance(elt, QuantizedArray) for elt in output_quantized_arrays)
 
-        return output_quantized_arrays[0].qvalues
+        results = tuple(
+            elt.qvalues for elt in output_quantized_arrays if isinstance(elt, QuantizedArray)
```

Comment:
is the `if isinstance(elt, QuantizedArray)` part really necessary ? I imagine this may be mypy but you have an assert for it right above

Reply:
yes type hint again